hledger/Hledger/Cli/CompoundBalanceCommand.hs

```diff
@@ -376,7 +376,7 @@ compoundBalanceReportAsHtml ropts cbr =
             Total simpleDateSpanCell totalrow
                              -- make a table of rendered lines of the report totals row
         & map (map (fmap wbToText))
-        & zipWith (:) (Spr.defaultCell "Net:" : repeat Spr.emptyCell)
+        & addRowSpanHeader (Spr.defaultCell "Net:")
```

Comment:
@thielema: At some point, this cell lost its "account" class, causing "Net" to no longer be left-aligned. I didn't see how exactly to fix it myself.

Reply:
By the way you may have seen this already but: adding `td, th {border: thin solid red;}` in a `hledger.css` in the current is quite handy for troubleshooting.